datadog_checks_base/tests/base/utils/http/test_authtoken.py

```diff
@@ -507,6 +507,55 @@ def fetch_token(self, *args, **kwargs):
 
             assert http.options['headers'] == expected_headers
 
+    def test_success_with_auth_params(self):
+        instance = {
+            'auth_token': {
+                'reader': {
+                    'type': 'oauth',
+                    'url': 'foo',
+                    'client_id': 'bar',
+                    'client_secret': 'baz',
+                    "auth_params": {
+                        "audience": "http://example.com",
+                        "scope": "openid"
+                    }
```

Comment:
```suggestion
                    "auth_params": {"audience": "http://example.com", "scope": "openid"},
```

This will fix the lint error in the CI. To format the code you can use `ddev test datadog_checks_base -fs` if you have [ddev](https://docs.datadoghq.com/developers/integrations/python/?tab=macos) installed

Reply:
To run this test, you can run `ddev test datadog_checks_base -- -k test_success_with_auth_params`, or you can rely on the CI if you do not want to install our CLI

Reply:
Fixed, thanks